internal/worker/apiserver/worker.go

```diff
@@ -62,6 +63,10 @@ type Config struct {
 	ObjectStoreGetter       objectstore.ObjectStoreGetter
 	ControllerConfigService ControllerConfigService
 	ModelService            ModelService
+
+	// ProviderFactory returns a provider for a given model. This should be
```

Comment:
I am not sure this comment is needed here as this is just dependency injection for the package.